sda-commons-client-jersey/src/test/java/org/sdase/commons/client/jersey/oidc/OidcClientTest.java

```diff
@@ -87,13 +85,7 @@ void before() throws IOException {
                 aResponse()
                     .withStatus(200)
                     .withHeader(CONTENT_TYPE, APPLICATION_JSON)
-                    .withBody(
-                        DW.getObjectMapper()
-                            .writeValueAsBytes(
-                                DW.getObjectMapper()
-                                    .readValue(
-                                        getClass().getResource("/fixtures/tokenResponse.json"),
-                                        new TypeReference<Map<String, Object>>() {})))));
+                    .withBody(fixture("fixtures/tokenResponse.json"))));
```

Comment:
Afaik, Wiremock has configuration for response body file locations … just in case you really want to change it.

Reply:
Thanks for the suggestion, I liked it better as well